guardrails/utils/openai_utils/streaming_utils.py

```diff
@@ -30,7 +31,7 @@ def num_tokens_from_messages(
     try:
         encoding = tiktoken.encoding_for_model(model)
     except KeyError:
-        print("Warning: model not found. Using cl100k_base encoding.")
```

Comment:
pretty sure the default and recommended logger formats will emit the lvls so we can let go of these Warning: in the messages